common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -93,7 +94,15 @@
     dirs = list(dirs)
     dirs.insert(0, theme.path / 'templates')
 engine = Engine(dirs=dirs)
-source, template_path = Loader(engine).load_template_source(path)
+loader = Loader(engine)
+
+for origin in loader.get_template_sources(path, dirs):
+    try:
+        source = loader.get_contents(origin)
+    except TemplateDoesNotExist:
+        pass
+raise TemplateDoesNotExist(path)
```

Comment:
Doesn't this always raise the exception, even if the template was found?  Need to check if `source` was assigned a value.